drivers/include/periph/flashpage.h

```diff
@@ -107,6 +107,27 @@ static inline int flashpage_page(void *addr)
 void flashpage_write(int page, void *data);
 
 /**
+ * @brief   Write the given address with the given data and length
+ *
+ * @note The given address must be aligned to the size of the minimal
+ *       writing block allowed by the MCU. This function doesn't erase
+ *       the block to be written automatically, so be sure the block
```

Comment:
I think this documentation can be/needs to be improved:
- it should be clear, that the `write_raw` can potentially write over block borders, so one write could effect multiple blocks
- put the long description outside the `@note` block
- just put a short warning into the `@note` block, as in "Make sure the targeted memory area is erased before calling this function" or something like this

Reply:
> put the long description outside the @note block

Where does this belong then?

Reply:
outside the @note block

Reply:
> just put a short warning into the @note block, as in "Make sure the targeted memory area is erased before calling this function" or something like this

If it's a warning I would suggest to rather use a [`@warning` block](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdwarning). ;-)